packages/twitch/src/API/Helix/Stream/HelixStream.ts

```diff
@@ -149,6 +149,16 @@ export class HelixStream {
 		return this._data.thumbnail_url;
 	}
 
+	/**
+	 * The URL of the thumbnail of the stream with a width and height.
+	 *
+	 * @param {number} width Width of the thumbnail.
+	 * @param {number} height Height of the thumbnail.
+	 */
+	getThumbnailUrl(width: number, hieght: number): string {
+		return this.thumbnailUrl.replace('{width}', width.toString()).replace('{height}', hieght.toString());
```

Comment:
Please don't use getters for accessing API data internally. Use the _data property instead.

Reply:
Also, the height parameter is misspelled.

Reply:
Fixed